### ISSUE TITLE
Luckysheet -> Univer

### ISSUE THREAD

**Issue by cvoges12** [divamgupta/obsidian-spreadsheets#23]
I'm not sure what changes were made between Luckysheet to Univer. But the Luckysheet repository is no longer developed because of some name change. Is there any changes that need to be made to this plugin to pull any changes since the last ones to Univer?

**Comment by divamgupta** [comment]
yes good idea, if anyone wants to send a PR, that would be appreciated
